src/electron/worker.js

```diff
@@ -38,9 +38,6 @@ class BitgoSingleton {
 
   // eslint-disable-next-line class-methods-use-this
   async [BitgoAction.RECONNECT]() {
-    if (process.env.NODE_ENV === 'test') {
-      return 'singleton created';
-    }
```

Comment:
@piggydoughnut this is the chunk of code that was preventing testcafe from connecting to the network, took me a while to find it 😅 

I think this just some leftover code since it didn't seem to have any effect on the app behavior, and jest also runs fine without it. But let me know if otherwise.

Reply:
eh, I think it was there because of the previous setup with C binaries or something. Its def not needed anymore :)